### PR TITLE
Update README.md and DEVELOPMENT.md

### PR DESCRIPTION
**Pull request by jeffoverflow**
Signed-off-by: quicksilver <zhifeng.zhang@zilliz.com>

/cc @scsven @xiaofan-luan

### CODE REVIEW

**Comment by scsven** [comment]
/cc @PahudPlus 
Please review the changes in README.md

**Comment by scsven** [comment]
The changes in DEVELOPMENT.md looks good to me.

File: README.md [hunk 1]
```diff
@@ -110,36 +103,11 @@ $ git clone https://github.com/milvus-io/milvus.git
...
-$ git clone https://github.com/milvus-io/milvus.git
-
-# Install third-party dependencies.
-$ cd milvus
-$ ./scripts/install_deps.sh
-
-# Compile Milvus Cluster.
-$ make milvus
-```
+For the full story, head over to the [developer's documentation](https://github.com/milvus-io/milvus/blob/master/DEVELOPMENT.md).
```
Comment (PahudPlus) [inline README.md:110]: head over to --> see

**Review by PahudPlus — CHANGES_REQUESTED** [review]

**Review by PahudPlus — APPROVED** [review]
/lgtm

**Review by PahudPlus — APPROVED** [review]

**Pull request merged**